ios/BT/API/Result.swift

```diff
@@ -31,17 +31,15 @@ public enum ExposureError: LocalizedError {
   case dailyFileProcessingLimitExceeded
   case cancelled
 
+  /// TODO localize this
```

Comment:
Is this comment still applicable? It looks like you've localized the strings below 😃

Reply:
I dont know how localization and react native works, I added this comment so we add the proper translations